MonoGame.Framework/Game.cs

```diff
@@ -65,7 +69,7 @@ public Game()
             _components = new GameComponentCollection();
             _content = new ContentManager(_services);
 
-            Platform = GamePlatform.Create(this);
+            Platform = GamePlatform.Create(this, embedContext);

```

Comment:
In case of non (WINDOWS && OPENGL) || LINUX || ANGLE platforms, embedContext is not defined here.